<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>cards</string>
    </property>
   </widget>
   <widget class="DropLabel" name="PlayerLabel1" native="true">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>410</y>
      <width>150</width>
      <height>225</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="DropLabel" name="PlayerLabel2" native="true">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>410</y>
      <width>150</width>
      <height>225</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="DropLabel" name="TableLabel1" native="true">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>150</width>
      <height>225</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="DropLabel" name="TableLabel2" native="true">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>150</width>
      <height>225</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="DropLabel" name="TableLabel3" native="true">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>60</y>
      <width>150</width>
      <height>225</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>380</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Your cards</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>30</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Table</string>
    </property>
   </widget>
   <widget class="DropLabel" name="TableLabel4" native="true">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>60</y>
      <width>150</width>
      <height>225</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="DropLabel" name="TableLabel5" native="true">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>60</y>
      <width>150</width>
      <height>225</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>610</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>580</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Players num</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>410</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string>Chances</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>460</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string>Win:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>510</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string>Tie:</string>
    </property>
   </widget>
   <widget class="QLabel" name="winLabel">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>460</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="TieLabel">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>510</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DropLabel</class>
   <extends>QWidget</extends>
   <header>droplabel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
